### DISCUSSION TITLE
Keyboard manipulation would be amazing.

### DISCUSSION THREAD

**Discussion by AlanEdd** [cxw42/TabFern#310] [General]
Hey there,
This is the best tab management solution by far. I used lots of different types and dumped eventually. 

1. **Simplicity, compact**
2. **reluctance to complicate** - Tabs outliner and TheBrain etc are unnecessarily complicated and cater to the enthusiast. This in it's current state caters to EVERYONE, it's just drowned out by the sheer number of competing extensions, emphasis on number and less emphasis on competing.

*The only thing to improve workflow is keyboard shortcuts -  Renaming, all the closing /saving functions / moving links up and down the list. etc would just supercharge organisation, rather than point and click.

*I combine it with a tab renamer extension - which is great, so I just rename the page, which eliminates the need for adding notes.
For example if I have "google docs - working title for 2023 docu on Trees.docx"  I just hit ctrl +r and rename the tab "Tree"; It makes TabFern look nice and tidy. 

**Anyway well done and thank you.**

**Comment by cxw42** [comment]
Thanks for the kind words, and for using TF!  I agree with you about keyboard shortcuts --- open as #28 .

**Comment by cxw42** [comment]
Which tab-renamer do you use?  I'd like to give that combination a try!  It occurs to me that "Add/edit a note" could reasonably get its own hotkey, so I have opened #311 for that.

**Reply by AlanEdd** [reply]
https://chrome.google.com/webstore/detail/simple-tab-renamer/ailhpmlejogfdcpoflidmobgkgdemaog?hl=en
I map to ctrl+r

![image](https://user-images.githubusercontent.com/36997610/214528395-a6f0aa77-1c58-4ed6-adb6-5d5aee2308a8.png)

![image](https://user-images.githubusercontent.com/36997610/214528549-665e379a-e1e9-4617-8f47-7b94d2572ace.png)
